elk/extraction/prompt_dataset.py

```diff
@@ -3,7 +3,7 @@
 from dataclasses import dataclass
 from datasets import DatasetDict, load_dataset
 from numpy.typing import NDArray
-from promptsource.templates import DatasetTemplates
+from promptsource_module.promptsource.templates import DatasetTemplates
```

Comment:
i think this was the main :feelsgood: 
basically locally we had `promptsource` installed. so the tests /typecheck would be okay.
On the pipeline, instead of `promptsource.templates`, we actually needed to call `promptsource.promptsource.templates`

anyways i've renamed it creatively to `promptsource_module` to save us from potential future headaches while we still use the submodule